downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -261,11 +261,11 @@ future<()> f1 = start runtime:sleep(50);
 
```

Comment:
For the above, how about just captioning the two code snippets as "Previous method" and "New method" ?

Reply:
`sleep` isn't new right? So we can't really say new method.

Reply:
Is it misleading? I meant method as in way. Not method as in function. :D

Reply:
At least for me it was :D Can we not just do https://github.com/ballerina-platform/ballerina-dev-website/pull/1071#discussion_r456560475

Reply:
+1 Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1072.